ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/EnginePayloadParameter.java

```diff
@@ -64,13 +64,13 @@ public EnginePayloadParameter(
       @JsonProperty("gasUsed") final UnsignedLongParameter gasUsed,
       @JsonProperty("timestamp") final UnsignedLongParameter timestamp,
       @JsonProperty("extraData") final String extraData,
-      @JsonProperty("receiptRoot") final Hash receiptsRoot,
+      @JsonProperty("receiptsRoot") final Hash receiptsRoot,
       @JsonProperty("logsBloom") final LogsBloomFilter logsBloom,
       @JsonProperty("prevRandao") final String prevRandao,
       @JsonProperty("transactions") final List<String> transactions,
       @JsonProperty("withdrawals") final List<WithdrawalParameter> withdrawals,
-      @JsonProperty("dataGasUsed") final UnsignedLongParameter dataGasUsed,
-      @JsonProperty("excessDataGas") final String excessDataGas,
+      @JsonProperty("blobGasUsed") final UnsignedLongParameter dataGasUsed,
+      @JsonProperty("excessBlobGas") final String excessDataGas,
```

Comment:
why is excessBlobGas a String? in the spec it's a quantity, same as blobGasUsed https://github.com/ethereum/execution-apis/blob/main/src/engine/cancun.md#ExecutionPayloadV3

Reply:
Wherever we are using json I have tried to keep the types used consistent with the other types in that same context.